observability-lib/Makefile

```diff
@@ -12,4 +12,8 @@ lint:
 
 .PHONY: test
 test:
-	go test -v ./...
\ No newline at end of file
+	go test -v ./...
+
+.PHONY: regenerate
+regenerate:
+	go test -v ./dashboards/... -generate=1
```

Comment:
Why `-v`?
```suggestion
	go test ./...

.PHONY: regenerate
regenerate:
	go test ./dashboards/... -generate=1
```

Reply:
nice catch forgot to remove the flag after debugging some issues thanks